ludwig/combiners/combiners.py

```diff
@@ -94,23 +122,52 @@ def output_shape(self) -> torch.Size:
         return output_tensor["combiner_output"].size()[1:]
 
 
+class BaseCombinerConfig(schema.BaseMarshmallowConfig):
+    """Base combiner config class."""
+
+    pass
+
+
 @dataclass
-class ConcatCombinerConfig:
+class ConcatCombinerConfig(BaseCombinerConfig):
+    """Parameters for concat combiner."""
+
     fc_layers: Optional[List[Dict[str, Any]]] = schema.DictList()
+    """TODO: Document parameters. (default: None)."""
+
     num_fc_layers: int = schema.NonNegativeInteger(default=0)
+    """TODO: Document parameters. (default: 0)."""
+
     output_size: int = schema.PositiveInteger(default=256)
+    """Output size of a fully connected layer (default: 256)."""
+
     use_bias: bool = True
+    """Whether the layer uses a bias vector (default: True)."""
+
     weights_initializer: Union[str, Dict] = schema.InitializerOrDict(default="xavier_uniform")
+    """TODO: Document parameters. (default: 'xavier_uniform')."""
```

Comment:
Can we remove TODOs from these docstrings, otherwise they will end up in generated UI elements.

Reply:
I was thinking to leave them here and then manually remove TODOs in UI elements.